### PR TITLE
Mid July; Engine deployments

### PR DESCRIPTION
**Pull request by jakerockland**
## Description of the change

Engine deployments for:

(goerli)
- https://github.com/ArtBlocks/artblocks/issues/574
- https://github.com/ArtBlocks/artblocks/issues/573
- https://github.com/ArtBlocks/artblocks/issues/1228
- https://github.com/ArtBlocks/artblocks/issues/1247
- https://github.com/ArtBlocks/artblocks/issues/1257

(mainnet)
- https://github.com/ArtBlocks/artblocks/issues/553
- https://github.com/ArtBlocks/artblocks/issues/552
- https://github.com/ArtBlocks/artblocks/issues/1203
- https://github.com/ArtBlocks/artblocks/issues/1246

### CODE REVIEW

File: packages/contracts/deployments/engine/V3/partners/tender-flex/MINTER_DEPLOYMENT_LOGS.log [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+}
+[ERROR] unable to allowlist the new minter on the minter filter at 0x3b90746ac1ccBFC5D6aC3a09d9930Dc3B224a0F9
+[INFO] this may be because the minter filter has already been transferred to a partner's wallet
+[INFO] Verifying MinterDAExpV4 contract deployment...
+The contract 0x582dE6f4c19De758D3A7242B9305f5a483Ad3579 has already been verified
+[INFO] MinterDAExpV4 contract verified on Etherscan at 0x582dE6f4c19De758D3A7242B9305f5a483Ad3579}
+[INFO] Minter deployment details written/appended to /Users/jakerockland/Code/artblocks-contracts/packages/contracts/deployments/engine/V3/partners/tender-flex/DEPLOYMENTS.md
+[INFO] Completed minter deployment! MinterDAExpV4 deployed to 0x582dE6f4c19De758D3A7242B9305f5a483Ad3579, and allowlisted on the minter filter at 0x3b90746ac1ccBFC5D6aC3a09d9930Dc3B224a0F9
+[ACTION] If this minter is to be indexed in a subgraph, ensure it is added to the subgraph's config.
+[ACTION] The minter was not able to be allowlisted on the minter filter, ensure the minter filter is updated by appropriate admin.
```
Comment (ryley-o) [inline packages/contracts/deployments/engine/V3/partners/tender-flex/MINTER_DEPLOYMENT_LOGS.log:82]: @jakerockland just checking that we gave the contract admin a heads-up for this 👍

File: packages/contracts/deployments/engine/V3/partners/tender/MINTER_DEPLOYMENT_LOGS.log [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+}
+[ERROR] unable to allowlist the new minter on the minter filter at 0xf377a477704DD68907BC603A7403B09B7b97DC0E
+[INFO] this may be because the minter filter has already been transferred to a partner's wallet
+[INFO] Verifying MinterDAExpV4 contract deployment...
+The contract 0x0283873C59ef310359a82a446508414F3b8D9C90 has already been verified
+[INFO] MinterDAExpV4 contract verified on Etherscan at 0x0283873C59ef310359a82a446508414F3b8D9C90}
+[INFO] Minter deployment details written/appended to /Users/jakerockland/Code/artblocks-contracts/packages/contracts/deployments/engine/V3/partners/tender/DEPLOYMENTS.md
+[INFO] Completed minter deployment! MinterDAExpV4 deployed to 0x0283873C59ef310359a82a446508414F3b8D9C90, and allowlisted on the minter filter at 0xf377a477704DD68907BC603A7403B09B7b97DC0E
+[ACTION] If this minter is to be indexed in a subgraph, ensure it is added to the subgraph's config.
+[ACTION] The minter was not able to be allowlisted on the minter filter, ensure the minter filter is updated by appropriate admin.
```
Comment (ryley-o) [inline packages/contracts/deployments/engine/V3/partners/tender/MINTER_DEPLOYMENT_LOGS.log:82]: (same callout)

**Review by ryley-o — APPROVED** [review]

File: packages/contracts/deployments/engine/V3/partners/tender-flex/MINTER_DEPLOYMENT_LOGS.log (hunk 1 above)
Comment (jakerockland) [reply packages/contracts/deployments/engine/V3/partners/tender-flex/MINTER_DEPLOYMENT_LOGS.log:82]: Yesser yesser – did do in the associated ticket tagging Dan. 😄

File: packages/contracts/deployments/engine/V3/partners/tender/MINTER_DEPLOYMENT_LOGS.log (hunk 2 above)
Comment (jakerockland) [reply packages/contracts/deployments/engine/V3/partners/tender/MINTER_DEPLOYMENT_LOGS.log:82]: Yesser yesser – did do in the associated ticket tagging Dan. 😄

**Pull request merged**